Base/usr/share/man/man8/useradd.md

```diff
@@ -0,0 +1,50 @@
+## Name
+
+useradd - add a new user to the system password file
+
+## Synopsis
+
+```**sh
+# useradd [options] <login>
+```
+
+## Description
+
+This program uses adds a new user to the system.
+
+By default, the user will be added to the **users** group (which has a GID of 100).
+
+This program must be run as root.
+
+## Options
+
+* `-u`, `--uid` _uid_: The user identifier for the new user. If not specified, an unused UID above `1000` will be auto-generated.
+* `-g`, `--gid` _gid_: The group identifier for the new user. If not specified, it will default to 100 (the **users** group).
+* `-s`, `--shell` _path-to-shell_: The shell binary for this login. The default is `/bin/Shell`. 
+* `-m`, `--create-home`: Create the specified home directory for this new user.
+* `-d`, `--home-dir` _path_: Set the home directory for this user to path). By default, this is `/home/username`, where `username` is the value of login.
```

Comment:
Still the stray `)` though :slightly_frowning_face:

Reply:
REEEEE